src/in-view.js

```diff
@@ -58,69 +66,76 @@ const inView = () => {
     * The main interface. Take a selector and retrieve
     * the associated registry or create a new one.
     */
-    let control = (selector) => {
-
-        if (typeof selector !== 'string') return;
-
-        // Get an up-to-date list of elements.
-        let elements = [].slice.call(document.querySelectorAll(selector));
+    let control = (selector, opts) => {
+
+        let elements;
+
+        if (typeof selector === 'string') {
+            // Get an up-to-date list of elements.
+            elements = [].slice.call(document.querySelectorAll(selector));
+        } else {
+            if (selector instanceof window.Node) {
+                elements = [selector];
+            } else if (selector instanceof window.NodeList || selector instanceof Array) {
```

Comment:
What use cases does `Array` cover? You'd have to do `instanceof jQuery` to catch those specifically unless you did `$('.foo').toArray()` first.

Reply:
You can do that or `$('.foo').get()`. The thing is, if you pass a jquery object, we'd need jquery here to take the elements out.